atc/db/task_queue.go

```diff
@@ -0,0 +1,153 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+
+	"code.cloudfoundry.org/lager"
+	sq "github.com/Masterminds/squirrel"
+)
+
+//go:generate counterfeiter . TaskQueue
+
+type TaskQueue interface {
+	FindOrAppend(string, string, int, string, lager.Logger) (int, int, error)
+	FindQueue(string) (string, int, string, error)
+	Dequeue(string, lager.Logger)
+	Length(string) (int, error)
+	Position(string) (int, error)
+}
+
+type taskQueue struct {
+	conn Conn
+}
+
+func NewTaskQueue(conn Conn) TaskQueue {
+	return &taskQueue{
+		conn: conn,
+	}
+}
+
+func (queue *taskQueue) FindOrAppend(id string, platform string, team_id int, worker_tag string, logger lager.Logger) (int, int, error) {
+	// Returns the position and the total queue length for a given id
+	var (
+		position int
+		len      int
+	)
+	ex_platform, ex_team_id, ex_worker_tag, err := queue.FindQueue(id)
```

Comment:
The code base (and golang in general) uses camel case for go variable names. Snake case gets used for database column names and queries.

```suggestion
	exPlatform, exTeamId, exWorkerTag, err := queue.FindQueue(id)
```

Reply:
I've replaced all snakes to camels :) :snake: :arrow_right: :camel: